requirements_dev.txt

```diff
@@ -1,5 +1,6 @@
 black
 pre-commit
+pylint
```

Comment:
it is strange that pylint is required. Did VScode complained it was not installed? If so that's maybe because the it is in the .vscode/settings.json. It is ok to remove it @crsegerie ?

Reply:
needed for one of the test packages.